test/parallel/test-dotenv-edge-cases.js

```diff
@@ -113,3 +113,35 @@ describe('.env supports edge cases', () => {
     assert.strictEqual(child.code, 0);
   });
 });
+
+// Ref: https://github.com/nodejs/node/pull/54913
+it('CLI edge cases', async () => {
+  const edgeCases = [
+    {
+      flags: [fixtures.path('empty.js'), '--env-file=nonexistent.env'],
+    },
+    {
+      flags: ['--eval=""', '--', '--env-file=nonexistent.env'],
+    },
+    {
+      flags: ['--invalid-argument', '--env-file=nonexistent.env'],
+      error: 'bad option: --invalid-argument',
+    },
+    {
+      flags: ['--env-file-ABCD=nonexistent.env'],
+      error: 'bad option: --env-file-ABCD=nonexistent.env'
+    },
+  ];
+  for (const { flags, error } of edgeCases) {
```

Comment:
These tests are too cryptic. Can you be more verbose with errors and actually assert what you are testing?

Reply:
I understand the tests–but I authored them, so I'm extremely biased. Can you give an example?

Reply:
What are you trying to test with the following flag? Why should it fail/succeed? How should it fail/succeed?

```
{
      flags: ['--eval=""', '--', '--env-file=nonexistent.env'],
    },
```